### PR TITLE
make notations of no-license more coherent

### PR DESCRIPTION
**Pull request by weakish**
- make notations of /no-license.md and licenses/no-license.html
  more coherent
- clarify only publishing code on a _public_ repo on github grants
  others the right to view and fork your code in
  licenses/no-license.html

### CODE REVIEW

File: no-license.md [hunk 1]
```diff
@@ -8,6 +8,6 @@ You're under no obligation to choose a license and it's your right not to includ
 
 You'll have to check with your own legal counsel regarding your particular project, but generally speaking, the absence of a license means that default copyright laws apply. This means that you retain all rights to your source code and that nobody else may reproduce, distribute, or create derivative works from your work. This might not be what you intend.
 
-Even in the absence of a license file, if you publish your source code in a public repository on GitHub, you have accepted the [Terms of Service](https://help.github.com/articles/github-terms-of-service) which do allow other GitHub users some rights. Specifically, you allow others to view and fork your repository. 
+Even in the absence of a license file, you may grant some rights in cases where you publish your source code to a site that requires accepting terms of service. For example, if you publish your source code in a public repository on GitHub, you have accepted the [Terms of Service](https://help.github.com/articles/github-terms-of-service) which do allow other GitHub users some rights. Specifically, you allow others to view and fork your repository.
```
Comment (afeld) [inline no-license.md:11]: @hoolio Are there sites that _don't_ require users to accept their Terms of Service?

**Comment by afeld** [comment]
Besides the question re: ToS, looks good to me!

File: no-license.md (hunk 1 above)
Comment (benbalter) [inline no-license.md:11]: Yes. Posting to your own site, S3, an FTP server, I'd assume some mega-upload type sites, etc.

Comment (afeld) [inline no-license.md:11]: To view, or to upload, or both?

Comment (benbalter) [inline no-license.md:11]: Both, but you could reword it that the dichotomy doesn't matter e.g., "if publishing your source code requires accepting a third-party TOS".

Comment (afeld) [inline no-license.md:11]: :+1: on @benbalter's suggested change

Comment (haacked) [inline no-license.md:11]: Seems to me the existing wording captures that conditional:

```
in cases where you publish your source code to a site that requires accepting terms of service
```

Note it's referring to these specific _cases_ where publishing to a site that requires accepting terms of service. Does it need to be clarified even more?

Comment (afeld) [inline no-license.md:11]: (͡๏̯͡๏)

I defer to others with betterer grammar.

**Comment by haacked** [comment]
Thanks for the submission!

thumbs-up-judo

**Pull request merged**